src/Microsoft.DotNet.CodeAnalysis/PackageFiles/codeAnalysis.props

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup>
```

Comment:
If we have such a condition, that might be worth it.

---
In reply to: [57484804](https://github.com/dotnet/buildtools/pull/583#discussion_r57484804) [](ancestors = 57484804)